packages/cx/project.json

```diff
@@ -22,6 +22,7 @@
             "executor": "workspace-plugins:visual-stories",
             "options": {
                 "themes": [
+                    "sap_horizon_hcb",
```

Comment:
Config is incorrect in this file. Output is `stories/Visuals/styles` instead of `stories/Visuals/cx`